v22.2/joins.md

```diff
@@ -133,13 +133,24 @@ Hash joins are performed on two tables as follows:
 
 ### Lookup joins
 
-The [cost-based optimizer](cost-based-optimizer.html) decides when it would be beneficial to use a lookup join. Lookup joins are used when there is a large imbalance in size between the two tables, as it only reads the smaller table and then looks up matches in the larger table. A lookup join requires that the right-hand (i.e., larger) table be indexed on the equality column. A [partial index](partial-indexes.html) can only be used if it contains the subset of rows being looked up.
+The [cost-based optimizer](cost-based-optimizer.html) decides when it would be beneficial to use a lookup join. Lookup joins are used when there is a large imbalance in size between the two tables, as it only reads the smaller table and then looks up matches in the larger table. A lookup join usually requires that the right-hand (i.e., larger) table be indexed on an equality column. A [partial index](partial-indexes.html) can only be used if it contains the subset of rows being looked up.
```

Comment:
I added "usually" because I read "the optimizer can perform a lookup join without an equality filter" as not involving an equality column. Not sure if I interpreted this correctly.